guides/working-remotely.md

```diff
@@ -0,0 +1,23 @@
+# Working remotely
+
+When a delivery team aren't all in the same place, it's important that we continue to communicate and collaborate as well as we can. This guide contains some suggestions about how to create a happy and productive remote working environment.
+
+## Coming and going
+
+When you arrive, go home, or leave your station for a while, let your teammates know in Slack.
+
+## Know the limits of your technology
+
+Dialing in to a video conference is usually fine for standups, whereas things like all-day workshops are more difficult. If video will be painful, consider attending by speakerphone, or not at all.
```

Comment:
can we use a different word instead of 'painful', for example 'technically challenging' ?

Reply:
I've removed it 😄

Reply:
fab. I'll merge then :D